core/src/main/java/com/scalar/db/api/OperationBuilder.java

```diff
@@ -281,6 +281,23 @@ interface ClearValues<T> {
     T clearValue(String columnName);
   }
 
+  interface ImplicitRreReadEnabled<T> {
+    /**
+     * Disable implicit pre-read for this put operation.
+     *
+     * @return the operation builder
+     */
+    T disableImplicitPreRead();
+
+    /**
+     * Sets whether implicit pre-read is enabled or not for this put operation.
+     *
+     * @param implicitRreReadEnabled whether implicit pre-read is enabled or not
+     * @return the operation builder
+     */
+    T implicitRreReadEnabled(boolean implicitRreReadEnabled);
```

Comment:
I've added the `implicitPreReadEnabled(boolean implicitPreReadEnabled)` method because it can be useful in some cases.

Let's see the following example:

```java
boolean implicitPreRead = ...;

Buildable buildable =
    Put.newBuilder()
        .namespace("ns")
        .table("tbl")
        .partitionKey(partitionKey)
        .clusteringKey(clusteringKey)
        .floatValue("c4", 1.23F)
        .doubleValue("c5", 4.56);

if (!implicitPreRead) {
  buildable.disableImplicitPreRead();
}

Put put = buildable.build();
```

With the `implicitPreReadEnabled(boolean implicitPreReadEnabled)` method, the above code can be rewritten as follows:

```java
boolean implicitPreRead = ...;

Put put =
    Put.newBuilder()
        .namespace("ns")
        .table("tbl")
        .partitionKey(partitionKey)
        .clusteringKey(clusteringKey)
        .floatValue("c4", 1.23F)
        .doubleValue("c5", 4.56)
        .implicitPreReadEnabled(implicitPreRead)
        .build();
```

So I think we need a method that takes a boolean parameter that toggles the implicit pre-read feature. However, we can discuss the method name (`implicitPreReadEnabled`) further if needed. Thank you.